encoders/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,100 @@
+log4j.additivity.org.apache.hadoop.hive.serde2.lazy.LazyStruct=false
+log4j.logger.org.apache.hadoop.hive.serde2.lazy.LazyStruct=OFF
+
+log4j.additivity.org.apache.hadoop.hive.metastore.RetryingHMSHandler=false
+log4j.logger.org.apache.hadoop.hive.metastore.RetryingHMSHandler=OFF
+
+log4j.additivity.hive.log=false
+log4j.logger.hive.log=OFF
+
+log4j.additivity.parquet.hadoop.ParquetRecordReader=false
+log4j.logger.parquet.hadoop.ParquetRecordReader=OFF
+
+log4j.additivity.org.apache.parquet.hadoop.ParquetRecordReader=false
+log4j.logger.org.apache.parquet.hadoop.ParquetRecordReader=OFF
+
+log4j.additivity.org.apache.parquet.hadoop.ParquetOutputCommitter=false
+log4j.logger.org.apache.parquet.hadoop.ParquetOutputCommitter=OFF
+
+log4j.additivity.hive.ql.metadata.Hive=false
+log4j.logger.hive.ql.metadata.Hive=OFF
+
+log4j.additivity.org.apache.hadoop.hive.ql.io.RCFile=false
+log4j.logger.org.apache.hadoop.hive.ql.io.RCFile=ERROR
+
+# Other Spark classes that generate unnecessary logs at INFO level
+log4j.logger.org.apache.spark.broadcast.TorrentBroadcast=WARN
+log4j.logger.org.apache.spark.ContextCleaner=WARN
+log4j.logger.org.apache.spark.MapOutputTracker=WARN
+log4j.logger.org.apache.spark.scheduler.TaskSchedulerImpl=WARN
+log4j.logger.org.apache.spark.storage.ShuffleBlockFetcherIterator=WARN
+log4j.logger.org.apache.spark.scheduler.DAGScheduler=WARN
+log4j.logger.org.apache.spark.scheduler.TaskSetManager=WARN
+log4j.logger.org.apache.spark.scheduler.FairSchedulableBuilder=WARN
+log4j.logger.org.apache.spark.scheduler.cluster.CoarseGrainedSchedulerBackend$DriverEndpoint=WARN
+log4j.logger.org.apache.spark.storage.BlockManagerInfo=WARN
+log4j.logger.org.apache.hadoop.hive=WARN
+# for all Spark generated code (including ad-hoc UnsafeProjection calls etc)
+log4j.logger.org.apache.spark.sql.catalyst.expressions.codegen.CodeGenerator=WARN
+log4j.logger.org.apache.spark.sql.execution.datasources=WARN
+log4j.logger.org.apache.spark.scheduler.SnappyTaskSchedulerImpl=WARN
+log4j.logger.org.apache.spark.MapOutputTrackerMasterEndpoint=WARN
+log4j.logger.org.apache.spark.MapOutputTrackerMaster=WARN
+log4j.logger.org.apache.spark.storage.memory.MemoryStore=WARN
+log4j.logger.org.apache.spark.MapOutputTrackerWorker=WARN
+log4j.logger.org.apache.parquet=ERROR
+log4j.logger.parquet=ERROR
+log4j.logger.org.apache.hadoop.io.compress=WARN
+log4j.logger.spark.jobserver.LocalContextSupervisorActor=WARN
+log4j.logger.spark.jobserver.JarManager=WARN
+log4j.logger.org.apache.spark.sql.hive.HiveClientUtil=WARN
+log4j.logger.org.datanucleus=ERROR
+# Task logger created in SparkEnv
+log4j.logger.org.apache.spark.Task=WARN
+log4j.logger.org.apache.spark.sql.catalyst.parser.CatalystSqlParser=WARN
+
+# for generated code of plans
+log4j.logger.org.apache.spark.sql.execution.WholeStageCodegenExec=DEBUG
+# for SnappyData generated code used on store (ComplexTypeSerializer, JDBC inserts ...)
+log4j.logger.org.apache.spark.sql.store.CodeGeneration=DEBUG
```

Comment:
Update this with the latest cluster/conf/log4j.properties.template. This now uses a separate file for generated code by default.

Reply:
Updated it.